### ISSUE TITLE
Visualizer enhancement for shadow models

### ISSUE THREAD

**Issue by deplanche** [xLightsSequencer/xLights#5040]
When using shadow models, it is typical that these shadow models get dragged and dropped onto the ports on the visualizer screen.  The user then has to go back to the actual model, set the properties to use start channel, and set the start channel for the start of the first shadow model.

It would be convenient if there was the ability to drag the actual model onto the shadow model that is already assigned to a port, and it would automagically force the real model to match the start channel of the model it was dropped onto.  I was thinking something like holding CTRL or ATL, when dragging and dropping onto the model would force this.

Not sure how to show in the visualizer that it was successful, but adding something there would be nice to verify.  But mainly just trying to avoid the extra windows and clicks.

**Comment by cybercop23** [comment]
Nope... you set the start channle of the actual model to be the start of the shaow model.. not chanel.. that way you never have to change it again.

**Comment by deplanche** [comment]
OK, mistyped there.  The actual model is set to be at the start of the model (in most cases, the start of the first shadow model).  Rather than having to open the dialog box for setting the start channel to be the start of the model, why couldn't this be done on the visualize screen? Basically locking the model to the shadow model, so that it follows it around.

**Comment by cybercop23** [comment]
You can't do it that way, since it doesn't know they are the same model or should be the same model. Maybe simply setting the Shadow Model For option can make the link, but I don't think we want to have any visualizer overlap for this fucntion.

**Comment by derwin12** [comment]
I don't think this is always the case .. thinking the shadow models we used for moving heads (pan/tilt/etc). Also other cases where you shadow model half a tree, one for each controller. There are likely other conditions where if we make it automagic, we break the flexability we have.
